src/ontology/mondo-edit.obo

```diff
@@ -559595,7 +559590,6 @@ subset: gard_rare {source="MONDO:GARD"}
 subset: rare
 xref: DOID:0081377 {source="MONDO:equivalentTo"}
 is_a: MONDO:0005336 {source="https://orcid.org/0000-0002-4142-7153"} ! myopathy
-is_a: MONDO:0009068 {source="DOID:0081377"} ! cytochrome-c oxidase deficiency disease
```

Comment:
@nicolevasilevsky I don't think this x-ref should have been removed. It should have been replaced by the "replaced by" ID.

Reply:
fixed